### ISSUE TITLE
Please consider a gem release

### ISSUE THREAD

**Issue by suweller** [m4i/rubocop-git#13]
Hi m4i,

I love `rubocop-git` and use it in `.git/hooks/pre-commit` in almost every ruby project. So first up, thanks so much for writing this gem!

I do have a small request though...
Commit e68d51d4a5567015466718c448eb3700b8f0d904 is needed in order to work in unison with rubocop 0.36.0 and 0.37.0.

This is the error message you get with the 0.0.5 release:
```
rubocop-0.37.0/lib/rubocop/processed_source.rb:23:in `initialize': wrong number of arguments (1 for 2..3) (ArgumentError)
rubocop-git-0.0.5/lib/rubocop/git/style_guide.rb:32:in `new'
rubocop-git-0.0.5/lib/rubocop/git/style_guide.rb:32:in `parse_source'
rubocop-git-0.0.5/lib/rubocop/git/style_guide.rb:14:in `violations'
rubocop-git-0.0.5/lib/rubocop/git/style_checker.rb:27:in `offenses'
rubocop-git-0.0.5/lib/rubocop/git/style_checker.rb:16:in `block in violations'
rubocop-git-0.0.5/lib/rubocop/git/style_checker.rb:15:in `map'
rubocop-git-0.0.5/lib/rubocop/git/style_checker.rb:15:in `violations'
rubocop-git-0.0.5/lib/rubocop/git/runner.rb:21:in `violations'
rubocop-git-0.0.5/lib/rubocop/git/runner.rb:52:in `display_violations'
rubocop-git-0.0.5/lib/rubocop/git/runner.rb:13:in `run'
rubocop-git-0.0.5/lib/rubocop/git/cli.rb:10:in `run'
rubocop-git-0.0.5/bin/rubocop-git:5:in `<top (required)>'
```

**Comment by pgaspar** [comment]
I've just run into the same issue.

**Comment by m4i** [comment]
Thank you.
I have released 0.0.6!

**Comment by suweller** [comment]
Awesome, thanks m4i!
